tools/util.ts

```diff
@@ -38,3 +39,35 @@ function findFilesWalk(paths: string[], depth: number) {
 
   return [].concat(...foundPaths);
 }
+
+export const executableSuffix = platform.os === "win" ? ".exe" : ""
+
+/**
+ * Returns true if the path exists.
+ */
+export function existsSync (path: string): boolean {
```

Comment:
Style error here - shouldn't be a space between existsSync and parens.

Reply:
Yes. Will fix it.

Something is wrong in this branch and formatting doesn't seem working now. I'll look into it.

Reply:
`std/prettier/main.ts` was crashing at `//tests/error_syntax.js` because the parser throws. The original prettier CLI seems ignoring the file when it can't parse it, and avoids crashing. I added `error_syntax.js` to skip list to avoid the crash.

Reply:
Because of this file: https://github.com/denoland/deno/blob/master/.prettierignore
I guess std/prettier doesn't look at it?